bolt-design-system/bolt
# Test Updating Jest VRT Diff Threshold

## Jira
http://vjira2:8080/browse/BDS-1050

## Summary
Testing out a config update to our Jest VRT settings to try and reduce the # of incorrectly failing VRT tests on Travis. 

## Details
Manually adds a new config option to the ratio and carousel Jest tests (the ones that seem to be failing the most) -- if this update works out, we'll likely want to move this VRT config up to the testing-helpers package to share globally.

## How to test
- Confirm Travis tests passing.
- Test locally running the full set of Jest VRT tests to kick the tires a bit more on this?
